src/vm-memory/Cargo.toml

```diff
@@ -5,6 +5,6 @@ authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2018"
 
 [dependencies]
-vm-memory-upstream = { package = "vm-memory", version = ">=0.2.2", features = ["backend-mmap"] }
+vm-memory-upstream = { package = "vm-memory",  version = "0.6.0", features = ["backend-mmap", "backend-bitmap"] }
```

Comment:
My take on this is that `create_guest_memory`  should be placed in vmm since this is the glue code needed in order to make use of upstream vm-memory so i do not see the need to have a separate crate in fc for this

Reply:
see this comment: https://github.com/firecracker-microvm/firecracker/pull/2661#issuecomment-896977110